src/index.html

```diff
@@ -1,12 +1,535 @@
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="catalog">
+        <div
+          data-qa="card"
+          class="card catalog__card"
+        >
+          <h1 class="visually-hidden">Product Card</h1>
+
+          <img
+            src="./images/imac.jpeg"
+            alt="Apple iMac A1419"
+            class="card__image"
+          >
+
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <div class="card__product-code">
+            Product code: 195434
+          </div>
+
+          <div class="card__reviews-wrapper">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="card__reviews-amount">
+              Reviews:
+              <span>5</span>
+            </div>
+          </div>
+
+          <div class="card__price-wrapper">
+            <div class="card__price-label">
+              Price:
+            </div>
+
+            <div class="card__price">
+              $2,199
+            </div>
+          </div>
+
+          <a
+            data-qa="card-hover"
+            href="#"
+            class="card__button"
+          >
+            Buy
+          </a>
+        </div>
+
+        <div class="card catalog__card">
+          <h1 class="visually-hidden">Product Card</h1>
+
+          <img
+            src="./images/imac.jpeg"
+            alt="Apple iMac A1419"
+            class="card__image"
+          >
+
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <div class="card__product-code">
+            Product code: 195434
+          </div>
+
+          <div class="card__reviews-wrapper">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="card__reviews-amount">
+              Reviews:
+              <span>5</span>
+            </div>
+          </div>
+
+          <div class="card__price-wrapper">
+            <div class="card__price-label">
+              Price:
+            </div>
+
+            <div class="card__price">
+              $2,199
+            </div>
+          </div>
+
+          <a
+            href="#"
+            class="card__button"
+          >
+            Buy
+          </a>
+        </div>
+
+        <div class="card catalog__card">
+          <h1 class="visually-hidden">Product Card</h1>
```

Comment:
hmmm, @Rijo-J why would you have multiple hidden H1's?

Reply:
If I'm not mistaken I think it was Misha's suggestion for screen readers and my card title has an h2 (or it was me who didn't understand it properly :confounded: most probably);
Should I remove the hidden headings completely and update card title h2 for h1?